examples/django/proj/celery.py

```diff
@@ -7,14 +7,16 @@
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'proj.settings')
 
+from django.conf import settings # noqa
+
 app = Celery('proj')
 
 # Using a string here means the worker will not have to
 # pickle the object when using Windows.
-app.config_from_object('django.conf:settings', namespace='CELERY_')
+app.config_from_object('django.conf:settings')
 
 # load task modules from all registered Django app configs.
-app.autodiscover_tasks()
+app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
```

Comment:
Did you revert this? It should be blank now since Django 1.8 apps may be virtual (AppConfigs)